xbmc/cores/VideoRenderers/BaseRenderer.cpp

```diff
@@ -225,7 +225,7 @@ RESOLUTION CBaseRenderer::FindClosestResolution(float fps, float multiplier, RES
 
   float fRefreshRate = fps;
 
-  float last_diff = fRefreshRate;
+  float last_diff = fRefreshRate*2;
```

Comment:
this changes behavior for all platforms. why is this hidden in this commit? why did you change it?